src/components/ConversationSettings/ModerationSettings.vue

```diff
@@ -91,25 +108,36 @@
 </template>
 
 <script>
+import Multiselect from '@nextcloud/vue/dist/Components/Multiselect'
 import { showError, showSuccess } from '@nextcloud/dialogs'
 import { CONVERSATION, WEBINAR } from '../../constants'
 import DatetimePicker from '@nextcloud/vue/dist/Components/DatetimePicker'
 import SipSettings from './SipSettings'
 
+const listableOptions = [
+	{ value: 2, label: t('spreed', 'Everyone') },
+	{ value: 1, label: t('spreed', 'Regular users, without guests') },
```

Comment:
Needs to be clear if its the guest app and to avoid confusion should only say registered/logged in users when guests is not there